packages/core/src/__tests__/rum/instrumentation/ShallowObjectEqualityChecker.test.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import { areObjectShallowEqual } from "../../../rum/instrumentation/ShallowObjectEqualityChecker";
+
+describe('areObjectValuesEqual', () => {
+    it('returns false for different object values', () => {
```

Comment:
Test is missing for the case when key is the same, but value has a different type. Not sure if this can happen, but it would be nice to have such.

Reply:
As we use the triple equality there should not be any issue, but this could be a good regression test, as for example `false == 0` returns `true`. I'm adding it.